src/stubs/ErrorUtils.js

```diff
@@ -11,13 +11,17 @@
 
 /* jslint unused:false */
 
-var ErrorUtils = {
-  applyWithGuard(callback, context, args, onError, name) {
-    return callback.apply(context, args);
-  },
-  guard(callback, name) {
-    return callback;
-  },
-};
+if (global.ErrorUtils) {
```

Comment:
This is the only change here that I can imagine potentially causing havoc on Facebook, I'll double check and report back. Should be fine, though.